client/src/components/prospects/Table.jsx

```diff
@@ -27,10 +27,33 @@ export default function ProspectsTable(props) {
 
   function fetchProspects() {
     Auth.getProspects().then((res) => {
-      setRows(res.data.prospects);
-      setFilteredRows(res.data.prospects);
+      let pList = res.data.prospects;
+      for (let i=0; i < pList.length; i++) {
```

Comment:
```suggestion
      pList = pList.map(p => {p.checked = false})
```

because you're only doing one action on each item, that may be a shorter code :)

Reply:
thanks Bonnie - i had to modify your suggestion slightly to this. It doesn't require the assignment operator (with it, it was returning a list of undefined) but this revision below is working now: 
```
      pList.map(p => {p['checked'] = false});

```